tests/OSPSuite.Infrastructure.Tests/OSPSuite.Infrastructure.Tests.csproj

```diff
@@ -17,7 +17,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="OSPSuite.BDDHelper" Version="4.0.1.1" />
-    <PackageReference Include="System.Data.SQLite.Core" Version="1.0.119" />
+    <PackageReference Include="Microsoft.Data.Sqlite" Version="8.0.0" />
```

Comment:
why are these all required? It was required before because we needed to copy the file "manually" so to speak I believe. Is this the same with this version? (e.g ther eis an OS specific file ?)